docs/network/mqtt.md

```
@@ -1,3 +1,21 @@
+## Home Assistant setup
+
+In Home Assistant
+
+* create non admin user in home assistant e.g mqttuser
+* install the Mosquitto MQTT Broker via Settings / Devices & Services, it will auto discover the MoonLight devices
+ 
+In MoonLight / Network / MQTT:
+
+* Enable MQTT
+* URI: mqtt://homeassistant.local:1883
+    * replace homeassistant.local by it appropriate mdns name or IP address
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)